toolchain/mfc/test/test.py

```diff
@@ -130,17 +130,18 @@ def test():
     exit(nFAIL)
 
 
-# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements, trailing-whitespace
 def _handle_case(case: TestCase, devices: typing.Set[int]):
     start_time = time.time()
 
-    tol = case.compute_tolerance()
+    if ARG("single"):
+        tol = 1e-5
```

Comment:
this doesn't make sense. i would instead use case.compute_tolerance() and then multiply it by 10^7 for single precision cases